lms/djangoapps/course_api/blocks/tests/test_api.py

```diff
@@ -208,7 +238,7 @@ def test_query_counts_cached(self, store_type, with_storage_backing):
             self._get_blocks(
                 course,
                 expected_mongo_queries=0,
-                expected_sql_queries=12 if with_storage_backing else 11,
+                expected_sql_queries=14 if with_storage_backing else 13,
```

Comment:
This number is bumped +2 because the transformer is behind a CourseWaffleFlag and is inactive for this test, so before assigning the default values, the underlying function will perform the query twice, one to course waffle flag table and one to waffle flag table([reference](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/waffle_utils/__init__.py#L234))